app/src/test/java/com/example/bottomnav/ParseTest.java

```diff
@@ -0,0 +1,171 @@
+package com.example.bottomnav;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+public class ParseTest {
```

Comment:
Consider having two tests for the const and struct case? The behavior is quite different, unified only by that they're both parsed from the same C source file.